### ISSUE TITLE
Problems with imagesize of 1024x1024

### ISSUE THREAD

**Issue by johanneskpp** [qubvel/segmentation_models#307]
I am using the U-Net-model. When i tried to set the input size to 1024x1024 the first 20 epochs had a validation iou-score of 1e-11. I had no problems with other sizes like 512x512, 256x256 and 768x768. I already tried another backbone. So why does this input-size not work?

**Comment by VladislavAD** [comment]
If it worked on other data it should work with other sizes too if the data is the same. Do you train multiclass segmentation? Have you checked that the data is correct and labels match input images? It take me ~300 epochs to gain first validation scores for my task, so 20 epochs looks a bit few for results. On the other hand if you use pretrained weights it should cover faster and I advice you to look for problems in input data.
